packages/protocol/contracts/common/FeeCurrencyWhitelist.sol

```diff
@@ -6,11 +6,26 @@ import "./interfaces/IFeeCurrencyWhitelist.sol";
 
 import "../common/Initializable.sol";
 
+import "../common/interfaces/ICeloVersionedContract.sol";
+
 /**
  * @title Holds a whitelist of the ERC20+ tokens that can be used to pay for gas
+ * Not including the native Celo token
  */
-contract FeeCurrencyWhitelist is IFeeCurrencyWhitelist, Ownable, Initializable {
+contract FeeCurrencyWhitelist is
+  IFeeCurrencyWhitelist,
+  Ownable,
+  Initializable,
+  ICeloVersionedContract
+{
+  // Array of all the tokens enabled
   address[] public whitelist;
+  // Array of all the non-Mento tokens enabled
+  address[] public nonMentoTokenWhitelist;
```

Comment:
Could we usr AddressSet for this ?

Reply:
Actually that variable is not needed anymore.